docs/concepts/queries/pull.md

```diff
@@ -36,7 +36,7 @@ Pull query features and limitations
   with a GROUP BY clause.
 - For non-windowed aggregations, pull queries only support looking up events
   by key.
-- WHERE clauses can only have constraints on the key column for non-windowed tables.
+- WHERE clauses can only have constraints on the key column(s) for non-windowed tables.

```

Comment:
Maybe add a line saying that all key columns must be present in the constraints? Unless we think that creates more confusion than it clears up.

Reply:
Seems like a good idea, esp. if we can show an example.